src/index.js

```diff
@@ -104,6 +110,7 @@ module.exports = window.AFRAME = {
   schema: require('./core/schema'),
   shaders: shaders,
   systems: systems,
+  ready: ready.ready,
```

Comment:
Do we need to make this API public?

Reply:
Yes, see the **Inline Module (w/ dynamic imports)** case. Basically we defer the initialization till after A-Frame is fully loaded + anything synchronous after that. However, if the user loads additional components/systems asynchronously after that point, they'll run into issues. In that case, and that case only, they'll need a way to signal when everything is ready. 

Note that this is also the original example motivating #5419.

Reply:
`ready` sounds like a flag. should we use a verb for a function. `init`, `start`?